src/frontend/src/pages/view-challenge/view-challenge.tsx

```diff
@@ -1,7 +1,91 @@
-import React from 'react';
+import React from "react";
+import Grid from "@material-ui/core/Grid";
+
+/* Text modules*/
+import Button from "@material-ui/core/Button";
+import { Link } from "react-router-dom";
+import Card from "@material-ui/core/Card";
+import CardContent from "@material-ui/core/CardContent";
+import Typography from "@material-ui/core/Typography";
+
+/* Check Box Modules */
+import Checkbox from "@material-ui/core/Checkbox";
+import { Box } from "@material-ui/core";
+
+import SubmitAnswerPopup from "./components/submit-answer-popup";
 
 export const ViewChallenge: React.FC = () => {
+    // TODO: We can add a state so that the component knows which box is selected
+    // Might have to make this a class that extends React.Component, I think React.fc
+    // doesn't allow for use of state
     return (
-        <div>View Challenge Page</div>
-    )
+        <Box p={2}>
```

Comment:
```suggestion
        <Container>
            <Box p={3}>
```
Maybe make the padding larger to be consistent with your grid. Also, if you wrap everything in a Container component, the width of the "page" won't exceed a certain max-width by default as I've previously recommended to @soroushMosaddad99